backuptar/tar_test.go

```diff
@@ -76,9 +76,9 @@ func TestRoundTrip(t *testing.T) {
 		t.Errorf("got size %d, expected %d", size, fi.Size())
 	}
 
-	if !reflect.DeepEqual(*bi, *bi2) {
-		t.Errorf("got %#v, expected %#v", *bi, *bi2)
+	if !reflect.DeepEqual(*bi2, *bi) {
```

Comment:
I'm not certain, but I think it was because I changed it in the error text and wanted to keep it consistent. The error text was backwards: `bi` is the expected value, `bi2` is the value we got from `FileInfoFromHeader`.

The original order `(expected, actual)` is more-consistent with something like `testify.assert.Equal` for this line, but `(actual, expected)` ordering is moire consistent with `==`.

Reply:
Okay. Thanks for the explanation.